src/MSALWrapper.Test/AuthModeTest.cs

```diff
@@ -64,14 +78,19 @@ public void OtherCombos()
             subject = AuthMode.Broker | AuthMode.Web;
             subject.IsBroker().Should().BeTrue();
             subject.IsWeb().Should().BeTrue();
+
+            subject = AuthMode.IWA | AuthMode.Broker;
+            subject.IsIWA().Should().BeTrue();
+            subject.IsBroker().Should().BeTrue();
         }
 
         [Test]
-        public void WebOrDeviceCodeIsNotbroker()
+        public void WebOrDeviceCodeIsNotBrokerAndIWA()

```

Comment:
Should this be `WebOrDeviceCodeIsNotBrokerOrIWA`? If so, could that be a separate test instead of mixing these two?

Reply:
It's the same thing. We are just validating that if we have `web` or `devicecode` as auth modes then broker and IWA will be false. Let me know your preference if we should use `or` or `and`.

Reply:
I think `or` makes more sense to me in this context if that's alright. 😃